### PR TITLE
Preload dotenv using node --require

### PR DESCRIPTION
**Pull request by lokankr**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -1,9 +1,4 @@
 /* istanbul ignore file */
-
-if (process.env.NEW_RELIC_ENABLED !== 'false') {
```
Comment (kibertoad) [inline src/server.ts:3]: how will it get executed in production, which does not use npm scripts?

File: jest.config.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "setupFiles": [
+    "<rootDir>/test/dotenvConfig.js"
```
Comment (kibertoad) [inline jest.config.json:3]: where does dotenvConfig.js come from?

File: package.json [hunk 3]
```diff
@@ -14,17 +14,17 @@
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint .  --ext .ts --fix",
     "format": "prettier --write . && npx prisma format",
-    "start:dev": "cross-env NODE_ENV=development NEW_RELIC_ENABLED=false ts-node src/server.ts",
-    "start:prod": "tsc && cross-env NODE_ENV=production node dist/src/server.js",
+    "start:dev": "cross-env NODE_ENV=development ts-node -r dotenv/config -r newrelic src/server.ts",
```
Comment (kibertoad) [inline package.json:17]: what if I want to disable newrelic locally?

Comment (lokankr) [reply package.json:17]: It ought to respect `NEW_RELIC_ENABLED` env var https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#agent-enabled

File: src/server.ts (hunk 1 above)
Comment (lokankr) [reply src/server.ts:3]: Could add it to `CMD` in `Dockerfile`.

Comment (kibertoad) [reply src/server.ts:3]: that would be difficult to delete for template users who don't use New Relic, as that will not show up during TS compilation, and can cause lots of headache.

**Review by kibertoad — APPROVED** [review]

**Pull request merged**